### ISSUE TITLE
Missing hover colors in high contrast

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#98]

**Comment by gabalafou** [comment]
When looking into this, try to find out if there are any discernible patterns, lessons learned, or rules we can apply in the future to ensure that future changes to the theme do not result is missing colors in high contrast mode.

**Comment by Carreau** [comment]
I don't have access to the audit report (which IMHO should be public), so would need a bit more context.

**Comment by trallard** [comment]
Agreed, this is a minor issue to focus on right now. We will revisit it later. For now, this is not planned.
